config_concept.yml

```diff
@@ -1,3 +1,6 @@
+Control:
```

Comment:
Do we intend to have multiple handlers (http, rmq) on same instance of macaw? Or each of them should be a dedicated instance?

Reply:
Not sure yet. But we can consider that